code/modules/reagents/newchem/medicine.dm

```diff
@@ -819,13 +817,88 @@ datum/reagent/stimulants/reagent_deleted(var/mob/living/M as mob)
 datum/reagent/insulin
 	name = "Insulin"
 	id = "insulin"
-	description = "Increases sugar depletion rates."
+	description = "A hormone generated by the pancreas responsible for metabolizing carbohydrates and fat in the bloodstream."
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 datum/reagent/insulin/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
-	if(M.sleeping)
-		M.sleeping--
```

Comment:
this was so that people who have been deep in a sugar coma can pop out of it a bit faster